src/SkprConfig.php

```diff
@@ -41,7 +41,10 @@ public function load(string $filename = self::DEFAULT_FILENAME): self {
       return $this;
     }
     if (empty($this->config)) {
-      clearstatcache(TRUE, $filename);
+      while (is_link($filename) || is_link(dirname($filename))) {
+        clearstatcache(TRUE, $filename);
+        $filename = dirname($filename) . '/' . readlink($filename);
```

Comment:
@acbramley the dirname bit feels a bit weird. What if the link is absolute instead of relative?

Reply:
Actually, we don't really need to generalise here. We know exactly what the symlink chain will be on skpr